ROADMAP.md

```diff
@@ -0,0 +1,51 @@
+# js-libp2p roadmap Q4’22/Q1’23
+
+## 🛣️ Milestones
+### 2022
+
+#### Early Q4 (October)
+- A.1📺 Universal Browser Connectivity - [**WebTransport**](https://github.com/libp2p/js-libp2p-webtransport/issues/1)
+
+#### End of Q4 (December)
+- A.2 📺 Universal Browser Connectivity - [**WebRTC**](https://github.com/little-bear-labs/js-libp2p-webrtc/pull/4) (Browser to Server)
+- B.1 🥊 Decentralized Hole Punching - [**AutoNat**](https://github.com/libp2p/js-libp2p/issues/1005)
+- B. 2 🥊 Decentralized Hole Punching - [**Circuit Relay v2**](https://github.com/libp2p/js-libp2p/issues/1029)
+
+### 2023
+
+#### End of Q1 (March)
+- B. 3 🥊 Decentralized Hole Punching - **Hole Punching**
+	- 🎉 Estimated Project Completion
+- C. 🧪 Future-proof Testing
+
+
+# Roadmap Appendix
+## A. 📺 Universal Browser Connectivity
+<!--- TODO: Link to GitHub Epic -->
+
+**Why**: A huge part of “the Web” is happening inside the browser. As a universal p2p networking stack, libp2p needs to be able to offer solutions for browser users.
+
+**Goal**: js-libp2p supports both WebTransport and (libp2p-) WebRTC protocols, enabled by default. This allows connections between browsers and public nodes, and eventually between browsers and non-public nodes and in between browser nodes.
+
+1. [WebTransport](https://github.com/libp2p/js-libp2p-webtransport/issues/1): Implementation of WebTransport in js-libp2p
```

Comment:
For these transport milestones, I think we should callout the done criteria is that their is a compatibility test with other implementations that runs as part of our release process.  I think we want stakeholders to know we're not just putting things out there but we're ensuring they interopate and will continue to.